src/main/java/calculator/InputHandler.java

```diff
@@ -0,0 +1,26 @@
+package calculator;
+
+import camp.nextstep.edu.missionutils.Console;
+
+public class InputHandler {
+    public static String getInput() {
+        System.out.println("덧셈할 문자열을 입력해 주세요.");
+        String input = Console.readLine();
+
+        checkNullInput(input);
+
+        return input;
+    }
+
+    private static void checkNullInput(String input) {
+        if (input == null || input.trim().isEmpty()) {
```

Comment:
isBlank를 사용하면 공백 검사를 더 쉽게 할수 있습니다!
```suggestion
        if (input == null || input.isBlank()) {
```